sculture-android/app/src/main/java/tr/edu/boun/cmpe/sculture/activity/TagActivity.java

```diff
@@ -0,0 +1,211 @@
+
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import tr.edu.boun.cmpe.sculture.R;
+
+import tr.edu.boun.cmpe.sculture.adapter.StoryListViewAdapter;
+import tr.edu.boun.cmpe.sculture.models.response.BaseStoryResponse;
+import tr.edu.boun.cmpe.sculture.models.response.SearchResponse;
+import tr.edu.boun.cmpe.sculture.models.response.TagResponse;
+
+import static tr.edu.boun.cmpe.sculture.BaseApplication.baseApplication;
+import static tr.edu.boun.cmpe.sculture.Constants.API_TAG_GET;
+import static tr.edu.boun.cmpe.sculture.Constants.API_USER_STORIES;
+import static tr.edu.boun.cmpe.sculture.Constants.BUNDLE_TAG_TITLE;
+import static tr.edu.boun.cmpe.sculture.Constants.FIELD_ID;
+import static tr.edu.boun.cmpe.sculture.Constants.FIELD_SIZE;
+import static tr.edu.boun.cmpe.sculture.Constants.FIELD_TITLE;
+import static tr.edu.boun.cmpe.sculture.Utils.addRequest;
+
+public class TagActivity extends AppCompatActivity {
+    private TextView tagTitle;
+    private TextView tagDescription;
+    private TextView editTagDesc;
+    private TextView save;
+    private TextView writer;
+    private TextView update_time;
+    private Button createDescription;
+    private RecyclerView story_list_recycler;
+
+    LinearLayoutManager mLayoutManager;
+    StoryListViewAdapter mStoryListViewAdapter;
+
+    private boolean is_looding_more = false;
+    private boolean is_reach_end = false;
+
+    private boolean is_on_edit = false;
+    private String previous = "";
+
+    private Activity mActivity;
+
+    Bundle bundle;
+    String tag_title;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.tag_description);
+        bundle = getIntent().getExtras();
+        if(bundle != null)
+            tag_title = bundle.getString(BUNDLE_TAG_TITLE);
+
+        Log.i("HERE", "" + tag_title);
+        tagTitle = (TextView) findViewById(R.id.tag_title);
+        tagDescription = (TextView) findViewById(R.id.tag_description);
+        writer = (TextView) findViewById(R.id.tag_creation);
+        update_time = (TextView) findViewById(R.id.tag_update);
+        editTagDesc = (TextView) findViewById(R.id.edit_cancel_text);
+        save = (TextView) findViewById(R.id.save_Text);
+        createDescription = (Button) findViewById(R.id.create_description);
+
+        story_list_recycler = (RecyclerView) findViewById(R.id.profile_story_list);
+        mLayoutManager = new LinearLayoutManager(getApplicationContext());
+        mStoryListViewAdapter = new StoryListViewAdapter(getParent());
+        story_list_recycler.setLayoutManager(mLayoutManager);
+        story_list_recycler.setAdapter(mStoryListViewAdapter);
+
+        getTag(tag_title);
+
+        setRecyclerListeners();
+
+        load_story();
+
+
+        if(tagDescription.getText().toString() != "") {
```

Comment:
If the tag has not any description, the view should be designed better. Last editor name, last edit date should be hidden. 
We can write some think like this with clickable TextView,
"This tag has not description. but you can write it"